services/s3/src/main/java/software/amazon/awssdk/services/s3/AwsS3V4Signer.java

```diff
@@ -192,13 +197,36 @@ private static long getContentLength(Aws4SignerRequestParams signerParams) throw
         return contentLength;
     }
 
-    @ReviewBeforeRelease("Create a builder for signers and set this in it. Also currently unexercised")
-    public void setDisableChunkedEncoding(boolean disableChunkedEncoding) {
-        this.disableChunkedEncoding = disableChunkedEncoding;
+
+    @NotThreadSafe
+    public interface Builder extends CopyableBuilder<Builder, AwsS3V4Signer> {
```

Comment:
I kind of like these just being classes instead of interfaces for most cases. What do you think?

Reply:
I was following the pattern we currently use in most classes. But I too like just having builder classes. Will change